.github/workflows/sync.yml

```diff
@@ -201,7 +201,7 @@ jobs:
           RCLONE_CONF: ${{ secrets.RCLONE_CONF }}
           RCLONE_GCS_SERVICE_ACCOUNT_CREDENTIALS: ${{ secrets.RCLONE_GCS_SERVICE_ACCOUNT_CREDENTIALS }}
         with:
-          args: "sync -c -v gs:cmip6 s3:cmip6-pds --exclude /{CMIP,ScenarioMIP,DCPP,GFDL_CM2_6,HighResMIP,CMIP5}/** --exclude /*.{bin,csv,csv.gz,html,json,jsonl} --transfers 15 --checkers 65"
+          args: "sync -c -v gs:cmip6 s3:cmip6-pds --exclude /{CMIP6,DCPP,GFDL_CM2_6,CMIP5}/** --exclude /*.{bin,csv,csv.gz,html,json,jsonl} --transfers 15 --checkers 65"
```

Comment:
I think this might need to be changed into a more advanced filter rule, as currently we are excluding all the other folders under `CMIP6/` from synchronization. In words, I think this job should be targeting:

- Folders under `CMIP6/` other than `CMIP/` and `ScenarioMIP`
- Folders under `gs://cmip6/` other than those already covered fully by other jobs

I think we also have another concurrent job free if we want to break this up into its own `CMIP6_remainder` job.